CHANGES.md

```diff
@@ -3,6 +3,8 @@
 #### next release (8.5.2)
 
 - Add `usePreCachedTilesIfAvailable` to `ArcGisMapServerCatalogItemTraits`.
+- Add `MergeStrategy` to `objectArrayTrait` - this includes a new `topStratum` strategy - similar to `Merge.All` (the default behaviour), but only elements that exist in the top-most strata will be merged with lower strata. Elements that only exist in lower strata will be removed.
```

Comment:
I think changing some of the traits to use these merge strategies should be listed as a breaking change.

Reply:
Good point, I hath added it as a breaking change in CHANGES